src/app/components/campus/campus.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+	<div class='campus-header' mymicds-blur>
+		<h1>Campus Life</h1>
+	</div>
+	<div class='campus-body'>
+		<div mymicds-blur class='subtitle'>
+			<h2>Community Service</h2>
+		</div>
+		<div mymicds-blur class='more'>
+			<p id='csclink'>
+			Do you want to get involved in your community? Do you want to volunteer at a variety of organizations around St. Louis? Visit this website to learn about all the different opportunities for MICDS Students or look below to see what other students have done for their communities!</p>
+			<br />
+			<p><a href='https://micdscsc.wixsite.com/opportunities' target='_blank' class='btn btn-success' role='button'>Community Service Committee Website</a></p>
```

Comment:
Would it be possible to center the button? IMO it just looks a little weird left-justified.